### PR TITLE
Fix Notify's mobile view

### PR DESCRIPTION
**Pull request by andrewoh531**
Changes can be viewed here - https://notify-f-mobile-ui.apps.y.cld.gov.au/features

### CODE REVIEW

File: admin/app/templates/govau_template.html [hunk 1]
```diff
@@ -53,6 +53,9 @@
               <span>{{ global_header_text|default('GOV.AU') }}</span>
               <span class="header-badge">alpha</span>
             </a>
+            <a href="#proposition-menu" id="open-navigation">Menu 
+              <img src="{{ asset_path }}images/navigation-open.svg" alt="Open navigation"/>
+            </a>
           </div>
           {% block inside_header %}{% endblock %}
         </div>
```
Comment (jonathaningram) [inline admin/app/templates/govau_template.html]: Not sure if you had planned to do it in this PR but can we have the home page look like the following?

image

File: admin/app/templates/admin_template.html [hunk 2]
```diff
@@ -43,14 +43,16 @@
 {% block proposition_header %}
   <div class="header-proposition">
     <div class="content">
-      <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
       <nav id="proposition-menu">
+        <a href="#" id="close-navigation">
```
Comment (jonathaningram) [inline admin/app/templates/admin_template.html]: ARIA?

File: admin/app/templates/govau_template.html [hunk 3]
```diff
@@ -53,6 +53,9 @@
               <span>{{ global_header_text|default('GOV.AU') }}</span>
               <span class="header-badge">alpha</span>
             </a>
+            <a href="#proposition-menu" id="open-navigation">Menu
```
Comment (jonathaningram) [inline admin/app/templates/govau_template.html]: ARIA?

File: admin/app/templates/govau_template.html (hunk 1 above)
Comment (andrewoh531) [reply admin/app/templates/govau_template.html]: Made some changes in this area. Let me know what you think @jonathaningram

**Comment by jonathaningram** [comment]
@andrewoh531 see:

image

Not sure if the changes in this PR caused the loss of left and right space on the above, but either way, can you take a look at fixing it as part of this PR?

**Comment by jonathaningram** [comment]
@dagoof 

> is this meant to be comprehensive or just for unauthenticated pages? once you start logging in it gets a bit weird

We have another ticket for auth pages.

**Pull request merged**